aiohttp/web_request.py

```diff
@@ -673,6 +673,18 @@ def body_exists(self) -> bool:
         self._post = MultiDictProxy(out)
         return self._post
 
+    def get_extra_info(self, name: str, default: Any = None) -> Any:
```

Comment:
I just saw that I defined the `name` parameter for `get_extra_info` to be a string both in the documentation and type hinting.
This is true, but it could also be any other valid dictionary key. Should I reconsider this?

Reply:
No, in asyncio the key is a string by convention.